templates/default/lsyncd.conf.lua.erb

```diff
@@ -1,9 +1,12 @@
 -- Managed by Chef
 
 settings = {
-  logfile = "<%= node['lsyncd']['log_file'] %>",
-  statusFile = "<%= node['lsyncd']['status_file'] %>",
-  statusInterval = <%= node['lsyncd']['interval'] %>
+  logfile = "<%= node['lsyncd']['settings']['logfile'] %>",
+  statusFile = "<%= node['lsyncd']['settings']['statusFile'] %>",
+  maxDelays = <%= node['lsyncd']['settings']['maxDelays'] %>,
+  maxProcesses = <%= node['lsyncd']['settings']['maxProcesses'] %>,
+  statusInterval = <%= node['lsyncd']['settings']['statusInterval'] %>,
+  insist = <%= node['lsyncd']['settings']['insist'] %>
```

Comment:
It would be more flexible if we just iterate on the `@settings` variables like this
```
<% setting = [] -%>
<% @settings.sort.each do |k, v| -%>
<% setting << "  #{v.to_s} = #{k.to_s}" %>
<% end -%>
<%= setting.join(",\n") %>
```
So the idea is to provide just a has with the options we want without constraints to the user. What do think about that?